src/worldfactory.cpp

```diff
@@ -1262,24 +1436,26 @@ int worldfactory::show_worldgen_tab_modselection( const catacurses::window &win,
 
                 startsel[0] = 0;
                 cursel[0] = 0;
+                recalc_start = true;
             }
         } else if( action == "NEXT_TAB" ) {
             tab_output = 1;
         } else if( action == "PREV_TAB" ) {
             tab_output = -1;
         } else if( action == "SAVE_DEFAULT_MODS" ) {
             if( mman->set_default_mods( active_mod_order ) ) {
-                popup( _( "Saved list of active mods as default" ) );
+                popup( _( "Saved list of active mods as default" ), PF_MOUSECTRL );
                 draw_modselection_borders( win, ctxt );
             }
         } else if( action == "VIEW_MOD_DESCRIPTION" ) {
             if( const MOD_INFORMATION *selmod = get_selected_mod() ) {
-                popup( "%s", mman_ui->get_information( selmod ) );
+                popup( mman_ui->get_information( selmod ), PF_MOUSECTRL );
```

Comment:
Using "%s" here is preferred because using a string directly as the first parameter to `popup` is ambiguous because `popup( char * )` and `popup( std::string )` behave differently (the first treats the string as a format string whereas the second shows the string as-is).